miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -468,7 +479,11 @@ Box.ui = {
   },
   
   discardOneItem: function() {
-    var selectedPosition = Box.utils.getPositionString(Box.visual.selected.row, Box.visual.selected.col);
+    if (Box.visual.selectedItems.length !== 1) {
+      alert('Select a single tube to discard');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)